dashboard/components/Layout.tsx

```diff
@@ -144,7 +144,7 @@ function Layout({ children }: { children: React.ReactNode }) {
           </Section>
           <Section>
             <NavTitle>Streaming</NavTitle>
-            <NavButton href="/dependency_graph/">Dependency Graph</NavButton>
+            <NavButton href="/dependency_graph/">Relation Graph</NavButton>
```

Comment:
I think the name "Relation Graph" doesn't inform users that they can find the dependency and backpressure information here. 🤔

I don't have very good idea also. What about "Relation Dependency and Backpressure Graph"..? Or simply "Relation Dependencies and Backpressures"?

Reply:
Oh, wait, we also have dependency and backpressure info in "Fragment Graph", right? Then "Relation Graph" should be good enough. 🤣